dev/com.ibm.ws.webcontainer/test/com/ibm/ws/webcontainer/webapp/SRTServletContainerVersionTest.java

```diff
@@ -88,52 +109,52 @@ public void testServlet31ObjectCreation() throws Exception {
+        //     }
+        // });
 
         Class<WebContainer> clazz = (Class<WebContainer>) webContainer.getClass();
 
-        Method versionSetter = clazz.getDeclaredMethod("setVersion", ServiceReference.class);
+        Field versionSetter = clazz.getDeclaredField("loadedContainerSpecLevel");
 
         versionSetter.setAccessible(true);
-        versionSetter.invoke(webContainer, versionRef);
+        versionSetter.set(webContainer, 30);
 
         assertTrue("Returned webApp container version should be 30", WebContainer.getServletContainerSpecLevel() == WebContainer.SPEC_LEVEL_30);
 
         context.assertIsSatisfied();
         ComponentMetaDataAccessorImpl.getComponentMetaDataAccessor().endContext();
     }
 
-    @SuppressWarnings("unchecked")
-    @Test
-    public void testServletErrorVersion() throws Exception {
-        WebContainer webContainer = new WebContainer();
-        final ServiceReference<ServletVersion> versionRef = context.mock(ServiceReference.class, "sr" + mockId++);
+    // @SuppressWarnings("unchecked")
+    // @Test
+    // public void testServletErrorVersion() throws Exception {
+    //     WebContainer webContainer = new WebContainer();
+    //     // final ServiceReference<ServletVersion> versionRef = context.mock(ServiceReference.class, "sr" + mockId++);
 
-        context.checking(new Expectations() {
-            {
+    //     // context.checking(new Expectations() {
+    //     //     {
 
-                allowing(versionRef).getProperty(ServletVersion.VERSION);
-                will(returnValue(WebContainer.SPEC_LEVEL_UNLOADED));
-            }
-        });
+    //     //         allowing(versionRef).getProperty(ServletVersion.VERSION);
+    //     //         will(returnValue(WebContainer.SPEC_LEVEL_UNLOADED));
+    //     //     }
+    //     // });
 
-        Class<WebContainer> clazz = (Class<WebContainer>) webContainer.getClass();
+    //     Class<WebContainer> clazz = (Class<WebContainer>) webContainer.getClass();
 
-        Method versionSetter = clazz.getDeclaredMethod("setVersion", ServiceReference.class);
+    //     Field versionSetter = clazz.getDeclaredField("loadedContainerSpecLevel");
 
-        versionSetter.setAccessible(true);
-        versionSetter.invoke(webContainer, versionRef);
+    //     versionSetter.setAccessible(true);
+    //     versionSetter.set(webContainer, -1);
 
-        assertTrue("Returned webApp container version should be 30", WebContainer.getServletContainerSpecLevel() == WebContainer.SPEC_LEVEL_30);
+    //     assertTrue("Returned webApp container version should be 30", WebContainer.getServletContainerSpecLevel() == WebContainer.SPEC_LEVEL_30);
 
-        context.assertIsSatisfied();
-        ComponentMetaDataAccessorImpl.getComponentMetaDataAccessor().endContext();
-    }
+    //     context.assertIsSatisfied();
+    //     ComponentMetaDataAccessorImpl.getComponentMetaDataAccessor().endContext();
+    // }
```

Comment:
SPEC_LEVEL_UNLOADED isn't really needed anymore (WC defaults to version 30 anyway).  I removed the variable, and I commented out the test for this reason.

Reply:
SPEC_LEVEL_UNLOADED made sense when it was loaded dynamically, but with the property file, I don't think we need it.